src/Shared/Microsoft.DotNet.Scaffolding.Shared/Project/ProjectModifierHelper.cs

```diff
@@ -31,7 +31,7 @@ internal static async Task<bool> IsMinimalApp(IModelTypesLocator modelTypesLocat
             if (startupType == null)
             {
                 //if changed the name in Program.cs, get the class name and check.
-                var programDocument = modelTypesLocator.GetAllDocuments().Where(d => d.Name.EndsWith("Program.cs")).FirstOrDefault();
+                var programDocument = modelTypesLocator.GetAllDocuments().FirstOrDefault(d => d.Name.EndsWith("Program.cs"));
```

Comment:
Turns out you can use a predicate inside of FirstOrDefault and then we don't need ".Where" so I replaced all those